xml/MAIN-SBP-CSP-UpdateInfra.xml

```diff
@@ -948,267 +1164,323 @@ regsharing:
-				starting the <package>rmt-server-regsharing</package> every thirty seconds to
-				synchronize registration information. </para>
-			<screen>
+  <sect2 xml:id="sec-reg-sharing-timer">
+   <title>Starting and Enabling the Registration Sharing Timer</title>
+   <para>
+    The systemd timer <package>rmt-server-regsharing.timer</package> defaults
+    to starting the <package>rmt-server-regsharing</package> every thirty
+    seconds to synchronize registration information.
+   </para>
+<screen>
 systemctl start rmt-server-regsharing.timer
 systemctl enable rmt-server-regsharing.timer
 					</screen>
-		</sect2>
-	</sect1>
-	<sect1 xml:id="sec-region-server-config-test">
-		<title>Region Server Configuration and Testing</title>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="sec-region-server-config-test">
+  <title>Region Server Configuration and Testing</title>
 
-		<sect2 xml:id="sec-region-server-config-test-intro">
-			<title>Introduction</title>
-			<para> The function of the Region Server is to provide information about the RMT servers
-				in a given region to a connecting client. The Region Server runs as a Python script
-				using the Flask framework in Apache. The Region Server is provided with the
-					<package>cloud-regionsrv</package> package. </para>
-			<para> Region servers are located in a CSP network. An example of the region server
-				placement could be based on geographic boundaries like Americas, Europe/Middle East
-				and Asia. It is recommended that a minimum of three region servers are deployed for
-				redundancy. Install, register and fully patch SUSE Linux Enterprise Server 15 SP1
-				for each of the region servers. </para>
-			<para> A client will attempt to contact a region server from a preconfigured list of
-				available region servers. The client, if configured and with an appropriate
-				implementation may provide a so called <emphasis role="strong">regionHint</emphasis>
-				to the region server. In the absence of a region hint the region server will use the
-				originating IP of the client instance to return a list of the closest RMT servers to
-				the client. </para>
-		</sect2>
+  <sect2 xml:id="sec-region-server-config-test-intro">
+   <title>Introduction</title>
+   <para>
+    The function of the Region Server is to provide information about the RMT
+    servers in a given region to a connecting client. The Region Server runs as
+    a Python script using the Flask framework in Apache. The Region Server is
+    provided with the <package>cloud-regionsrv</package> package.
+   </para>
+   <para>
+    Region servers are located in a CSP network. An example of the region
+    server placement could be based on geographic boundaries like Americas,
+    Europe/Middle East and Asia. It is recommended that a minimum of three
+    region servers are deployed for redundancy. Install, register and fully
+    patch SUSE Linux Enterprise Server 15 SP3 for each of the region servers.
```

Comment:
SLES version specific

Reply:
There are other places in the document where SUSE Linux Enterprise Server 15 SP3 is used.  I am not going to change the SLES version specific information because this shows the reader that the document is being updated for newer versions.